spring-petclinic-microservices/pom.xml

```diff
@@ -38,6 +38,8 @@
     <webjars-jquery.version>3.1.1-1</webjars-jquery.version>
     <webjars-angular.version>1.6.4</webjars-angular.version>
     <webjars-angular-ui-router.version>1.0.30</webjars-angular-ui-router.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```

Comment:
Why is this necessary?

Reply:
It's used by [spring-boot-maven-plugin](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/spring-cloud-azure_4.5.0/spring-petclinic-microservices/pom.xml#L128).

Without it, will compile error: 

![image](https://user-images.githubusercontent.com/5254035/209553713-5f6249ae-5b31-46f2-a483-bc8b9de895c9.png)

------

Why we didn't define it before?

Because our project inherit from `spring-boot-starter-parent` at first. These 2 properties defined there, check it in [spring-boot-starter-parent pom](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-starter-parent/2.7.7/spring-boot-starter-parent-2.7.7.pom).